### PR TITLE
Add js files to module Folder

### PR DESCRIPTION
**Pull request by LYANGEND**
**In this milestone  have achieved the following**;

- Keep all functionalities of the app without errors.
- Divide the code into modules. 
- Refactor my methods and functions to arrow functions.
- Use let and const in a correct way.

### CODE REVIEW

File: modules/create-book.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  const removeButton = document.createElement('button');
+  removeButton.className = 'remove-button';
+  removeButton.textContent = 'Remove';
+  bookTitle.textContent = book.title;
+  bookAuthor.textContent = book.author;
+  authorContainer.append(bookTitle, span, bookAuthor);
+  bookContainer.append(authorContainer, removeButton);
+  booksContainer.appendChild(bookContainer);
+};
+export default createBook;
```
Comment (keroloussamy) [inline modules/create-book.js]: - [ ] Kindly when I tried to add a book, nothing happened. Kindly fix it.

File: main.js [hunk 2]
```diff
@@ -11,93 +10,15 @@ class Book {
...
 const contactSection = document.querySelector('.contact');
 const displayBookSection = document.querySelector('.top-container');
-
 // Date
 const date = document.querySelector('.date');
-const dateTime = luxon.DateTime.utc().toLocaleString(luxon.DateTime.DATETIME_FULL);
-date.textContent = dateTime;
+setInterval(() => {
+  date.innerHTML = DateTime.now().toLocaleString(DateTime.DATETIME_MED_WITH_SECONDS);
+}, 1000);
```
Comment (keroloussamy) [inline main.js]: - [ ] Kindly add the current date below the navigation bar, to match the requirements.

image

File: main.js [hunk 3]
```diff
@@ -11,93 +10,15 @@ class Book {
...
-    form.elements.author.value = '';
-  }
-}
-
 const mainPage = new MainUI();
-const store = new Store();
-
 const addBookSection = document.querySelector('.bottom-container');
 const contactSection = document.querySelector('.contact');
 const displayBookSection = document.querySelector('.top-container');
```
Comment (keroloussamy) [inline main.js]: - [ ] Kindly fix the main content, to be just displaying one section per click on the menu.

image

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @LYANGEND, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Check the comments under the review._

- [ ] [OPTIONAL] Kindly change your files names, to `index.html`, `index.css`, `index.js`, to fully match the requirements.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/create-book.js (hunk 1 above)
Comment (Kingstalux) [reply modules/create-book.js]: resolved 👍

File: main.js (hunk 2 above)
Comment (Kingstalux) [reply main.js]: resolved 👍

File: main.js (hunk 3 above)
Comment (Kingstalux) [reply main.js]: resolved 👍

**Review by Kingstalux — APPROVED** [review]
Hi @LYANGEND,
# Status: Approved ✔️ ✔️ ✔️ 
 
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job implementing the requested changes 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- [ ] (OPTIONAL) Your delete function does not work well. It would be great if you could give each book a unique `id` and use that unique `id` to delete them. This will fix the issues when deleting a book 👍 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**